src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js

```diff
@@ -76,12 +76,20 @@ export default View.extend({
     `,
   getTemplateData() {
     const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
+    let enrolledDeviceName = '';
+    if (contextualData && contextualData?.devicebootstrap && contextualData?.devicebootstrap.enrolledDevices) {
+      const enrolledDevices = contextualData?.devicebootstrap.enrolledDevices;
+      enrolledDeviceName = Array.isArray(enrolledDevices) && !_.isEmpty(enrolledDevices) ?
+        enrolledDevices[0] : enrolledDevices;
```

Comment:
I assume `enrolledDevices` needs to be a string, but based on this check

```
Array.isArray(enrolledDevices) && !_.isEmpty(enrolledDevices)
```

if `enrolledDevices` is an empty array, enrolledDevices would equal `[]`
Secondly, if `devicebootstrap.enrolledDevices` is a non-array, is it guaranteed to be a string?

Reply:
`enrolledDevices` is expected to be an array of one string, but if there are more, we need to pick the first one. if just a string provided, that's also ok, eventually we need to pass to the `arguments` on line 62 either an array of one string, or just a string